recipe/meta.yaml

```diff
@@ -8,25 +8,28 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 43ce60d05ffa442ba9d3c5722193aadb93c38174b9fb471c8ea7ccbf8349bbca
+  sha256: fad87d13696127ffb27ea0882d6182f1a9cf8a5e2b37a587751166c51e5a332a
 
 build:
   noarch: python
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
wouldn't it be good to keep the `--no-deps` just in case?

Reply:
it doesn't matter much because ``conda-build`` disables the installation of dependencies via ``pip``
https://github.com/conda-forge/staged-recipes#7-when-or-why-do-i-need-to-use--python---m-pip-install---vv

but in our other docs we do have
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=python#use-pip

:smile: